### ISSUE TITLE
Prajwal Krishna 

### ISSUE THREAD

**Issue by prajwal2495** [hil-se/fds#3]
Hey All,

I am from India living in Bengaluru right now.
Well as far as Covid situation is concerned I am sure it has become a new normal for all of us.

Amidst all this I am very excited to start this Master's journey with RIT and absolutely gutted that we can not be present in Rochester physically and have fun together. I am sure we all get to meet in Spring 2021 ( !2020 ) :D

Data science is a very new thing for me and I am sure the foundations of data science will put all of us on the right track.
Let us all collaborate together and make the most of the facilities given by RIT for remote learning.

MyGithub: prajwal2495

**Comment by azhe825** [comment]
@prajwal2495 
 - Please click on the _Labels_ tag on the right side bar and select _intro_. 
 - After that, remove yourself from the _Assignees_.

This is a good way to learn how to interact on github.

**Comment by prajwal2495** [comment]
@azhe825 
Hey Professor,

I believe only contributors can add labels to an issue.
I was not given any option to add a label while creating and I tried to add by editing and I could not.

**Comment by azhe825** [comment]
@prajwal2495 
Thanks for letting me know. Looking for solutions.

**Comment by azhe825** [comment]
@prajwal2495 
I would assume that it has been fixed by creating multiple issue templates.
Can you try opening a new issue using the intro template?
I will close this one once it is done.

**Comment by prajwal2495** [comment]
Hey Professor,
@azhe825 
I just created a new issue.
Yes, it got fixed by creating the multiple issue templates.
Thank you.
